File: Day 013/1823. Find the Winner of the Circular Game.js
/**
 * @param {number} n
 * @param {number} k
 * @return {number}
 */
var findTheWinner = function(n, k) {
    a = 0;
        for(i = 1; i <= n; i++)
        {
            a = (a + k) % i;
        }
        return a + 1;
};

File: Day 013/1823. Find the Winner of the Circular Game.py
class Solution(object):
    def findTheWinner(self, n, k):
        a =0
        for i in range (1, n+1):
            a= (a+k)%i
        return a+1
        """
        :type n: int
        :type k: int
        :rtype: int
        """